# HG changeset patch
# User Rik <rik@octave.org>
# Date 1545609370 28800
#      Sun Dec 23 15:56:10 2018 -0800
# Branch stable
# Node ID 9254cdba20b269f76f913acc6680a18f80041e71
# Parent  24d8411954c8a7d55f7725d7923f99d4485c3e16
doc: Update subtitle Edition number in octave.texi.

* octave.texi: Update edition number to 5.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -110,17 +110,17 @@ modified versions.
 @author David Bateman
 @author S@o{}ren Hauberg
 @author Rik Wehbring
 @page
 @vskip 0pt plus 1filll
 Copyright @copyright{} 1996, 1997, 1999, 2000, 2001, 2002, 2005, 2006,
 2007, 2011, 2013, 2015, 2016, 2017, 2018 John W. Eaton.
 
-This is the fourth edition of the Octave documentation, and is consistent
+This is the fifth edition of the Octave documentation, and is consistent
 with version @value{VERSION} of Octave.
 
 Permission is granted to make and distribute verbatim copies of
 this manual provided the copyright notice and this permission notice
 are preserved on all copies.
 
 Permission is granted to copy and distribute modified versions of this
 manual under the conditions for verbatim copying, provided that the entire
